src/GameView.js

```diff
@@ -0,0 +1,29 @@
+export default class GameView {
+  /** @type { (bool) => void } */
+  static ToggleGameRestartButton(shouldShow) {
+    const gameRestartButton = document.querySelector('#game-restart-button');
+
+    if (shouldShow === true) {
+      gameRestartButton.style.display = 'block';
+    } else if (shouldShow === false) {
+      gameRestartButton.style.display = 'none';
+    }
```

Comment:
```suggestion
    if (shouldShow) {
      gameRestartButton.style.display = 'block';
      return;
    } 
    gameRestartButton.style.display = 'none';
    
```

얼리 리턴과, boolean값을 생략하는 방향으로 수정하면 훨씬 간결해질 것 같은데 
건님의 생각은 어떠한가요?

Reply:
boolean 값은 생략하는게 더 보기 좋을 것 같습니다!
개인적인 취향인데, 저는 개인적으론 다음과 같은 if 문에서는 else if를 사용하는 것을 선호합니다.
변수가 boolean 이기 때문에 else if 대신 else 를 사용해도, 혜정님이 제안하신 대로 언리 리턴을 사용해도 모두 같은 결과를 보여주지만, else if 로 정확한 조건을 명시하는 것이 저는 좀 더 가독성이 좋은 것 같습니다. 
또한 추가로 저 if문과 상관 없이 공통적으로 호출해야 하는 코드를 작성할 수도 있으니 지금의 방식이 저는 좋아 보입니다! 제 고집일까요?

Reply:
코드 스타일의 차이며 각각의 장단점이 있으므로 정답은 없다고 생각합니다. 
건님께서 말씀해주신 

>if문과 상관 없이 공통적으로 호출해야 하는 코드를 작성할 수도 있다

는 점을 생각하며 지금의 방식이 더 적합할 수 있어 보이네요!